packages/account/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Routes from 'Containers/routes.jsx';
+import ResetTradingPassword from 'Containers/reset-trading-password.jsx';
+import { MobxContentProvider } from 'Stores/connect';
+import initStore from 'Stores/init-store';
+import { TCoreStore } from 'Stores/index';
+
+type TAppProps = {
+    passthrough: {
+        root_store: TCoreStore;
+        WS: Record<string, any>;
```

Comment:
Can we add a TODO here? when we finished working on store's types we can change these

Reply:
good point Niloofar :+1: